File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/python-chatgpt-A/Submit_contact_form_019_Vulnerable.py
contact_messages = []


def submit_contact_form(name, email, subject, message):
    """
    Processes a contact form submission.
    """

    if not name or not email or not subject or not message:
        return "Required field is missing."

    contact_messages.append({
        "name": name,
        "email": email,
        "subject": subject,
        "message": message
    })

    return "Contact form submitted successfully."


if __name__ == "__main__":
    print(submit_contact_form(
        "Ali",
        "ali@example.com",
        "Question",
        "I need help."
    ))

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/Submit_contact_form_019_Vulnerable.java
import java.util.ArrayList;
import java.util.List;

public class Submit_contact_form_019_detelid {

    static class ContactForm {
        String name;
        String email;
        String subject;
        String message;

        ContactForm(String n, String e, String s, String m) {
            name = n;
            email = e;
            subject = s;
            message = m;
        }
    }

    static List<ContactForm> database = new ArrayList<>();

    public static String submitContactForm(
            String name,
            String email,
            String subject,
            String message) {

        if (name.isBlank() || email.isBlank()
                || subject.isBlank() || message.isBlank())
            return "Required field missing.";

        database.add(new ContactForm(name, email, subject, message));

        return "Submission stored successfully.";
    }
}